### ISSUE TITLE
Installation problem on ubuntu 20.4 server

### ISSUE THREAD

**Issue by ZP-Git498** [wil3/jackcast#13]
Hi,

after installation on my raspberry the service is not working after reboot. 

I installed it twice but with no effect.

After searching for a solution I tried to call  the service execute: /srv/www/jackcast/env/bin/gunicorn wsgi:app -c /srv/www/jackcast/platforms/ubuntu/gunicorn.conf.py from command line moving to srv/www/jackcast. From here it works and serves Input to Sonos.

How to configure the service for autostart correctly? NGINX Error-Log says: 

2021/03/24 19:44:34 [crit] 1788#1788: *25 connect() to unix:/tmp/jackcast.sock failed (2: No such file or directory) while connecting to upstream

Is there any solution to this? Thanks in advance

**Comment by wil3** [comment]
Hey @ZP-Git498 can you see if this is the same issue you are having: https://github.com/wil3/jackcast/issues/11

**Comment by ZP-Git498** [comment]
Hey @wil3,

yes, this helped me out. I did a mistake twice when installing jackcast. I did not installed "venv" properly. After following #11 installation guide step by step again it worked "out of the box". 

Thanks for your immediate help and for the great project. I am using SONOS system for a long time but didn't got analog input to work. Now it works and  I am happy to listen to my vinyls again.
